### PR TITLE
Add article explaining dashboard approaches, their advantages and dis…

### PR DESCRIPTION
**Pull request by kandl**
…advantages

- Add summary of all approaches for building and embedding GoodData dashboards
- List pros and cons for each approach

JIRA: RAIL-4056

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1015/ : SUCCESS in 2m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/677/ : SUCCESS in 6s

File: docs/02_choose_dashboard_approach.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+GoodData.UI offers multiple ways to build or embed your dashboard solution each with their advantages and disadvantages.
+This article should help you understand how they differ and help you decide which of them is appropriate for your particular use case.
+
+
+
+## Kpi Dashboards
+KpiDashboards is GoodData product that allows you to build interactive dashboards from your measures and insights with friendly drag and drop experience.
+It offers lot of features out of the box (filtering, drilling, sharing, responsive layout, theming, e-mail schedules, exports, embedding).
+You can embed these dashboards in you application with [iframe](#iframe-embedding) and also extend the functionality of the dashboard with [Dashboard Plugins](#dashboard-plugins).
```
Comment (ivanmjartan) [inline docs/02_choose_dashboard_approach.md:16]: Maybe is good to mention postMessage API too if there is plugins

File: docs/02_choose_dashboard_approach.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+id: choose_dashboard_approach
+---
+
+GoodData.UI offers multiple ways to build or embed your dashboard solution each with their advantages and disadvantages.
+This article should help you understand how they differ and help you decide which of them is appropriate for your particular use case.
+
+
+
+## Kpi Dashboards
+KpiDashboards is GoodData product that allows you to build interactive dashboards from your measures and insights with friendly drag and drop experience.
```
Comment (lupko) [inline docs/02_choose_dashboard_approach.md:14]: KPI Dashboards is a GoodData end-to-end dashboarding solution and covers both creation and consumption of the dashboards. Creators can build or edit existing dashboards with a friendly drag and drop experience.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1016/ : SUCCESS in 1m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/678/ : SUCCESS in 7s

File: docs/02_choose_dashboard_approach.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+#### Pros:
+- Totally under your control - custom visualization types, custom layout, styling, etc...
+- Huge amount of SDK React components and hooks ready to use (both high and low level - [charts](#start_with_visual_components), [execution](#create_new_visualization), [placeholders](#placeholders), contexts)
+- [Accelerator Toolkit](#create_new_application)
+- [Catalog Export](#export_catalog)
+- Seamless integration with existing React applications
+- Allows deepest integration with your application(s), with huge amount of possibilities.
+- Backend agnostic API that supports both Enterprise and GoodData.CN editions.
+- TypeScript, documented interfaces and API(s).
```
Comment (no23reason) [inline docs/02_choose_dashboard_approach.md:91]: We should add that you could also use a different framework if you wanted and still use the sdk-backend-* packages to make life easier.

File: docs/02_choose_dashboard_approach.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+#### Cons:
+- Plugins are currently supported in view mode only.
+- Plugins api is still a bit limited in comparison with full [Dashboard Component](#dashboard-component) capabilities.
+
+---
+
+## Dashboard Component
+
+Dashboard component is highly customizable React component, with lot of built-in features, that displays dashboards created with [Kpi Dashboards](#kpi-dashboards) in the exact same way - it basically an engine, that [Kpi Dashboards](#kpi-dashboards) use under the hood.
```
Comment (no23reason) [inline docs/02_choose_dashboard_approach.md:63]: in the exact same way as what, I feel there is a part of the sentece missing

File: website/i18n/en.json [hunk 5]
```diff
@@ -2113,6 +2117,10 @@
         "title": "Access Low Level API (DataLayer)",
         "sidebar_label": "Access Low Level API"
       },
+      "version-8.9.0/version-8.9.0-choose_dashboard_approach": {
```
Comment (no23reason) [inline website/i18n/en.json:2120]: We probably do not want this in 8.9.0, right?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1017/ : SUCCESS in 1m 56s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/679/ : SUCCESS in 6s

File: website/i18n/en.json (hunk 5 above)
Comment (kandl) [reply website/i18n/en.json]: removed

File: docs/02_choose_dashboard_approach.md (hunk 4 above)
Comment (kandl) [reply docs/02_choose_dashboard_approach.md]: updated

File: docs/02_choose_dashboard_approach.md (hunk 2 above)
Comment (kandl) [reply docs/02_choose_dashboard_approach.md]: updated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1028/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/687/ : SUCCESS in 7s

**Comment by evgeniabogranskaya** [comment]
Proofread
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1033/ : SUCCESS in 1m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/690/ : SUCCESS in 7s

**Review by plavemar — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR506

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1034/ : SUCCESS in 1m 32s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/213/ : FAILURE in 2m 30s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/108/ : SUCCESS in 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/303/ : SUCCESS in 21s

**Pull request merged**